images/image-predefined-group/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 The Kubernetes Authors.
```

Comment:
nit:
```suggestion
# Copyright 2022 The Kubernetes Authors.
```

can be fixed-up later on as well :+1:

Reply:
OK. I will fix up this in https://github.com/kubernetes-sigs/cri-tools/pull/1005.